src/surfaces/helpers-surface.php

```diff
@@ -103,8 +103,28 @@ public function __construct(
 	 * @return mixed The helper class.
 	 */
 	public function __get( $helper ) {
+		return $this->container->get( $this->get_helper_class( $helper ) );
+	}
+
+	/**
+	 * Magic isset for ensuring helper exists.
+	 *
+	 * @param string $helper The helper to get.
+	 *
+	 * @return bool The helper class.
+	 */
+	public function __isset( $helper ) {
+		return $this->container->has( $this->get_helper_class( $helper ) );
+	}
+
+	/**
+	 * Get the class name from a helper slug
+	 *
+	 * @param string $helper
+	 * @return string
+	 */
+	protected function get_helper_class( string $helper ) {
```

Comment:
The `string` class here isn't `use`d. This means that PHP defaults to the namespace this class is in rather than the global namespace.

Reply:
It has been removed